components/servo/lib.rs

```diff
@@ -296,6 +301,16 @@ fn create_constellation(user_agent: Cow<'static, str>,
                         layout_thread::LayoutThread,
                         script::script_thread::ScriptThread>::start(initial_state);
 
+    if cfg!(target_os = "windows") {
```

Comment:
It's just because HTC Vive only works on Windows now (no other SDKs implemented yet). I'm going to add a preference for this instead of platform filtering.